ceph_cfg/ops_cluster.py

```diff
@@ -58,3 +58,73 @@ def status_refresh(self):
                         output["stderr"])
                         )
         self.model.cluster_status = json.loads(output["stdout"].strip())
+
+
+    def _df_node_to_osd_dict(self, node):
+        node_type = node.get("type")
+        if node_type != "osd":
+            return None
+        osd_id = node.get("id")
+        osd_name = node.get("name")
+        osd_weight_crush = node.get("crush_weight")
+        osd_weight = node.get("reweight")
+        osd_kb_total = node.get("kb")
+        osd_kb_used = node.get("kb_used")
+        osd_kb_avail = node.get("kb_avail")
+        osd_pgs = node.get("pgs")
+        osd_utilization = node.get("utilization")
+        return {
+            "id" : osd_id,
```

Comment:
I would agree, either directly construct in return statement or list comprehension/dict comprehension if keys stay the same. Direct construction would make key change obvious enough I guess.